src/tactic/move_add.lean

```diff
@@ -0,0 +1,245 @@
+/-
+Copyright (c) 2022 Arthur Paulino, Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Arthur Paulino, Damiano Testa
+-/
+import tactic.core
+
+/-!
+# `move_add`: a tactic for moving summands
+
+Calling `move_add [a, ← b, c]`, recursively looks inside the goal for expressions involving a sum.
+Whenever it finds one, it moves the terms `a, b, c`, removing all parentheses.  The terms preceded
+by a `←` get placed to the left, the ones without the arrow get placed to the right.  Unnamed terms
+stay in place.  Due to re-parenthesizing, doing `move_add` with no argument may change the goal.
+Also, the *order* in which the terms are provided matters: the tactic reads them from left to right.
+This is especially important if there are multiple matches for the typed terms in the given
+expressions.
+
+### Warning:
+* The tactic uses lemmas from `algebra.group.basic`, thus it will not work unless this file is
+  imported
```

Comment:
Is there a reason you can't just import this file? Or put the tactic in that file.

Reply:
No reason, I simply thought that since the file compiled without it, there would have been an `unused imports`-linter!  I added the import.